shell/phpbrew.fish

```diff
@@ -347,6 +348,12 @@ function __phpbrew_update_config
     else
         set VERSION $argv
     end
+
+    if [ ! -d "$PHPBREW_ROOT/php/$VERSION" ]
+        echo "php version: $VERSION not exists."
```

Comment:
Looks like we're missing the same in `bashrc`. Could you add this logic there as well?